webapp_settings/production.py

```diff
@@ -53,11 +53,7 @@
 
 CHECK_HOST_DIRTY = True
 DIRTY_FILE_PATH = "~/server_dirty"
-{% if wcivf_controller|default(False) %}
-EE_BASE = "https://elections.democracyclub.org.uk"
-{% else %}
 EE_BASE = "http://localhost:8000"
```

Comment:
Now that the controller will also be serving requests, I can see why you want to use the local EE for serving web traffic. Does it become problematic that the local copy of EE may be up-to ~1 day behind when running management commands? Could this cause problems if we are (for example) receiving candidates for elections we don't know about yet?

Reply:
It might cause problems, but not the one you're thinking about. _Everything_ in WCIVF is imported from YNR (well, apart from the few random CSVs/publicly editable Google Sheets we import from), so we're not relying on EE as a source of anything apart from Postcode->Election lookups.

I think being up to 24 hours behind on that is _ok_ (how many people search for an election that quickly after we make it?), but even if it weren't ok, I think this is a different issue about making sure EE is always up to date / imports / updates work well.

Reply:
> we're not relying on EE as a source of anything apart from Postcode->Election lookups.

Right, well.. nearly :) but looking at this code:

https://github.com/DemocracyClub/WhoCanIVoteFor/blob/c149903dae8c05278c73c5f08ce255f932e419d0/wcivf/apps/elections/management/commands/import_elections.py#L30-L32

we'll just `pass` if we hit something we don't know about so this is non-critical.

Reply:
Ah, good catch, I'd forgotten about that! :)